core/src/main/scala/com/avast/grpc/jsonbridge/ReflectionGrpcJsonBridge.scala

```diff
@@ -0,0 +1,151 @@
+package com.avast.grpc.jsonbridge
+
+import cats.effect.Async
+import cats.syntax.all._
+import com.avast.grpc.jsonbridge.GrpcJsonBridge.GrpcMethodName
+import com.google.common.util.concurrent.{FutureCallback, Futures, ListenableFuture}
+import com.google.protobuf.util.JsonFormat
+import com.google.protobuf.{Message, MessageOrBuilder}
+import com.typesafe.scalalogging.StrictLogging
+import io.grpc.MethodDescriptor.{MethodType, PrototypeMarshaller}
+import io.grpc._
+import io.grpc.inprocess.{InProcessChannelBuilder, InProcessServerBuilder}
+import monix.eval.Task
+import monix.execution.Scheduler
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{Future, Promise}
+import scala.language.higherKinds
+import scala.util.control.NonFatal
+
+class ReflectionGrpcJsonBridge[F[_]](services: ServerServiceDefinition*)(implicit val scheduler: Scheduler, protected val F: Async[F])
+    extends GrpcJsonBridge[F]
+    with AutoCloseable
```

Comment:
Since we're already on the new Monix and thus cats-effect, I suggest we do this via [`cats-effect`'s `Resource`](https://typelevel.org/cats-effect/datatypes/resource.html) instead of Java's `AutoCloseable`

Reply:
btw, probably the best way to do that would be to make the constructor private and create a factory method in a companion object returning `Resource[F, ReflectionGrpcJsonBridge]`

Reply:
It's IMHO out of scope of this PR. This should have been done during update to the new Monix/cats-effect.

Please prepare a new PR for this change, thanks!

Reply:
OK, I will make a PR for this
given that, I think we're ready to merge (and also please to the `0.8.x` branch) (modulo those 2 other small changes I've proposed)